src/osip-model/src/test/java/edu/kit/pse/osip/core/model/behavior/FillAlarmTest.java

```diff
@@ -41,7 +41,7 @@ public void init() {
     public void testAlarmIsNotTriggered() {
         testLiquid = new Liquid(20f, 300f, new Color(0.5, 0.5, 0.5));
         tank = new Tank(200f, TankSelector.valuesWithoutMix()[0], testLiquid, pipe1, pipe2);
-        alarm = new FillAlarm(tank, 50f, AlarmBehavior.GREATER_THAN);
+        alarm = new FillAlarm(tank, 0.5f, AlarmBehavior.GREATER_THAN);
```

Comment:
Finde ich nicht nötig und auch nicht so gut, weil die Liquids in jedem Test neu (und unterschiedlich) erzeugt werden. Denn wenn man die Alarmschwelle ändert, muss man auch oft die Liquids anpassen.